soh/soh/config/ConfigUpdaters.cpp

```diff
@@ -53,5 +53,11 @@ namespace LUS {
             CVarSetInteger("gZFightingMode", CVarGetInteger("gDirtPathFix", 0));
             CVarClear("gDirtPathFix");
         }
+        if (CVarGetInteger("gRandomizedEnemies", 0) != 0) {
+            if (CVarGetInteger("gSeededRandomizedEnemies", 0)) {
+                CVarSetInteger("gRandomizedEnemies", 2);
+            }
+            CVarClear("gSeededRandomizedEnemies");
+        }
```

Comment:
Both of these variables existed before this PR right? It's sounding to me like we want to clear `gSeededRandomizedEnemies` at the end here regardless of what value `gRandomizedEnemies` happens to be.
```suggestion
        if (CVarGetInteger("gRandomizedEnemies", 0) != 0) {
            if (CVarGetInteger("gSeededRandomizedEnemies", 0)) {
                CVarSetInteger("gRandomizedEnemies", 2);
            }
        }
        CVarClear("gSeededRandomizedEnemies");
```

It may be that the way this value is handled that it would already effectively get removed, but there's no harm in being thorough, if the variable isn't there then that `CVarClear` will just do nothing.